db/fixtures/lessons/foundation_lessons.rb

```diff
@@ -81,18 +81,18 @@ def foundation_lessons
       identifier_uuid: '8a13a21d-453e-4492-8240-58a2c8d9596f',
       installation_lesson: true,
     },
-    'Fundamentals Part 1' => {
-      title: 'Fundamentals Part 1',
-      description: "You'll get a chance to start picking up the programming fundamentals you need to make your webpages dynamic",
+    'Variables and operators' => {
+      title: 'Variables and operators',
+      description: "In this lesson we learn about variables and operators.",
```

Comment:
```suggestion
      description: 'In this lesson we learn about variables and operators.',
```
Need this string in single quotes to make the linter happy.